uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/LeaveRequestCalendarFeedConfig.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>LeaveRequestCalendarFeedConfig</class>
+  <name>civicrm_hrleaveandabsences_calendar_feed_config</name>
+  <comment>Leave Request Calendar Feed Config</comment>
+  <add>4.4</add>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique LeaveRequestCalendarFeedConfig ID</comment>
+    <add>4.4</add>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>title</name>
+    <type>varchar</type>
+    <length>127</length>
+    <required>true</required>
+    <unique>true</unique>
+    <comment>The Calendar Feed Config Title. There cannot be more than one entity with the same title</comment>
+  </field>
+
+  <field>
+    <name>timezone</name>
+    <type>varchar</type>
+    <length>50</length>
+    <required>true</required>
+    <comment>The Calendar Feed Config Timezone</comment>
+  </field>
+
+  <field>
+    <name>hash</name>
+    <type>varchar</type>
+    <length>50</length>
```

Comment:
The size of an md5 hash is 32 chars. Why is this field size 50?

Reply:
I agree. Just left it like that incase we change the logic later and we might need more chars. I could make it exactly 32 anyway.